src/languageServerInstaller.ts

```diff
@@ -96,8 +78,36 @@ export class LanguageServerInstaller {
 		});
 	}
 
-	removeOldBinary(directory: string, platform: string): void {
-		if (platform === "windows") {
+	goOs(): string {
```

Comment:
I would probably not add to the Class definition and just call the functions directly. Enforcing style in JS is much more subjective so I tend to go with the flow, but in general I almost never use Classes and opt for more "pure functions and passed objects". So my subjective bias is to lean away from the class use.

Reply:
Fair point. 👍🏻  I will move it into functions.
Funny enough I did think about that possibility, but I "went with the flow" as you say 😄